instrumentation/redis/lib/opentelemetry/instrumentation/redis/instrumentation.rb

```diff
@@ -20,6 +20,7 @@ class Instrumentation < OpenTelemetry::Instrumentation::Base
         end
 
         option :peer_service, default: nil, validate: :string
+        option :enable_statement_obfuscation, default: false, validate: :boolean
```

Comment:
```suggestion
        option :enable_statement_obfuscation, default: true, validate: :boolean
```

Reply:
@robertlaurin I've added a commit with this